README.md

```diff
@@ -1,13 +1,14 @@
 # Project Mongo API
 
-Replace this readme with your own information about your project.
+This was my second backend project and the first using Mongo DB and Mongoose.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+## The challenge
 
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I used the basecode from my previous project https://github.com/MarianneArdin/project-express-api
 
 ## View it live
+https://wk18mongoapi.herokuapp.com/
+Endpoints: /goldenglobes
+/nominations
```

Comment:
Looks like you have more endpoints in your code. Perhaps you can add them here also :)